Automattic/jetpack
# Bootstrap connection status into My Jetpack

Adds the connection status card to the My Jetpack page. 

  ![image](https://user-images.githubusercontent.com/746152/145567656-259b0dae-74a6-42e4-869f-69182af286ea.png)










#### Jetpack product discussion


p9dueE-3Ae-p2 and p1HpG7-dFw-p2#comment-51138

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Checkout this branch, or launch a JN site with it.
* Build Backup plugin (`jetpack build plugins/backup`)
* Build my-jetpack package (`jetpack build packages/my-jetpack`)
* Add define( 'JETPACK_ENABLE_MY_JETPACK', true ); to your wp-config
* Confirm Jetpack backup is active and connected.
* Visit the My Jetpack Menu item under Jetpack  
  ![image](https://user-images.githubusercontent.com/746152/145567949-8b11c13e-79bb-4eb8-97fe-db8e7d3de2d7.png)
* Confirm you see the Connection status card like in the screenshot above.
* NOTE: The Stauts card will only be displayed when the site is connected

Testing the Disconnect redirect.
* Once disconnected you should always be redirected to the first page below the Jetpack top level menu item. (See discussion in p1HpG7-dFw-p2#comment-51138)
* Test disconnecting the site with different Jetpack plugins activated, and verify that you are always redirected to whatever is the at the top
* If you want, play with the `position` arguments used in menu registration to make sure we always catch the first.
